### PR TITLE
Prepare new msgraph

### PR DESCRIPTION
**Pull request by ivy-jla**

### CODE REVIEW

**Comment by ivy-jla** [comment]
Hello @ivy-rew 
Could you check if those changes are correct? Thanks :)

**Comment by ivy-rew** [comment]
There is maven build + jenkinsfile in the 'market/connector' directory. 
It seems to me as if we should delete it right now, as the 'ms-graph-demo' was the only reason, this build still resided there.

File: market/connector/microsoft365/calendar/meta.json [hunk 1]
```diff
@@ -1,13 +1,28 @@
 {
-	"id": "ms-calendar",
+	"id": "msgraph-calendar",
 	"version": "1.0",
```
Comment (ivy-rew) [inline market/connector/microsoft365/calendar/meta.json:3]: version is no longer interpreted anywhere I think, so I'd remove it.

File: market/connector/microsoft365/calendar/meta.json [hunk 2]
```diff
@@ -1,13 +1,28 @@
...
-	"sourceUrl": "https://github.com/axonivy/market",
+	"sourceUrl": "https://github.com/axonivy-market/msgraph-connector",
 	"language": "English",
 	"industry": "Cross-Industry",
-	"compatibility": "9.2+",
-	"tags": ["office"]	
+	"compatibility": "9.4+",
+	"tags": ["office"],
+	"mavenArtifacts": [
+		{
```
Comment (ivy-rew) [inline market/connector/microsoft365/calendar/meta.json:14]: please adapt from an existing sibling ... e.g. docuSign:
- ZIP product artifact is missing
- use the same in the 'mavenArtifacts' order (1. zip, 2. connector, 3. demo)

**Review by ivy-rew — CHANGES_REQUESTED** [review]

File: market/connector/microsoft365/calendar/meta.json (hunk 2 above)
Comment (ivy-rew) [reply market/connector/microsoft365/calendar/meta.json:14]: I guess the same applies for the other meta.json files

**Comment by alexsuter** [comment]
> There is maven build + jenkinsfile in the 'market/connector' directory. It seems to me as if we should delete it right now, as the 'ms-graph-demo' was the only reason, this build still resided there.
> 
> Sample applies for the docker-infrastructure there ... we don't need it anymore

Yes. And then please delete the build on Jenkins itself.
https://jenkins.ivyteam.io/view/scrum-master/job/market-connectors/

File: market/connector/microsoft365/toDo/meta.json [hunk 3]
```diff
@@ -13,16 +12,23 @@
 	"mavenArtifacts": [
 		{
 			"repoUrl": "https://maven.axonivy.com",
-			"name": "Microsoft Graph ToDo Demo",
+			"name": "Microsoft Graph Product",
 			"groupId": "com.axonivy.connector.office365",
-			"artifactId": "msgraph-todo-demo"
+			"artifactId": "msgraph-connector-product",
```
Comment (ivy-rew) [inline market/connector/microsoft365/toDo/meta.json:17]: haven't we spent so much time to craft 4 distinct products? ... shouldn't we then also consume unique products in each of these meta.json files?

**Review by ivy-rew — CHANGES_REQUESTED** [review]

File: market/connector/microsoft365/toDo/meta.json (hunk 3 above)
Comment (ivy-jla) [reply market/connector/microsoft365/toDo/meta.json:17]: 2 comments up you said I should iclude the ZIP. 
should I not do it then?

**Review by ivy-rew — APPROVED** [review]
Thanks, Looks good to me.

**Pull request merged**